cmd/command.go

```diff
@@ -594,6 +594,27 @@ func hIncrBy(cli *Client, args [][]byte) (interface{}, error) {
 	return cli.db.HIncrBy(key, field, incrInt64Val)
 }
 
+func hRandfield(cli *Client, args [][]byte) (interface{}, error) {
```

Comment:
As a suggestion:
```suggestion
func hRandField(cli *Client, args [][]byte) (interface{}, error) {
```

Reply:
I have a feeling you pointed this out last time too:sweat:
Thanks for the suggestion!